introspection-engine/introspection-engine-tests/tests/tables/mod.rs

```diff
@@ -299,10 +307,15 @@ async fn default_values(api: &TestApi) -> crate::TestResult {
                     t.add_column("int_static", types::integer().default(2).nullable(true));
                     t.add_column("float_static", types::float().default(1.43).nullable(true));
                     t.add_column("boolean_static", types::boolean().default(true).nullable(true));
-                    t.add_column(
-                        "datetime_now",
-                        types::datetime().default(functions::current_timestamp()).nullable(true),
-                    );
+
+                    if sql_family.is_postgres() {
+                        t.inject_custom("datetime_now Timestamp Default current_timestamp");
```

Comment:
```rust
t.add_column("datetime_now"), types::timestamp().default(functions::current_timestamp()))
```

Reply:
There is no timestamp type, but time works as well for the purposes of this test.

Reply:
We could maybe change this to be `timestamp` here, then we wouldn't need this `if`: https://github.com/prisma/barrel/blob/mssql-support/src/backend/pg.rs#L219